### PR TITLE
245: Saved search saving error

### PR DESCRIPTION
**Pull request by webdev-dp**
#245

### CODE REVIEW

**Comment by webdev-dp** [comment]
@dvesh3 here is new PR to 5.0

File: src/Model/SavedSearch/Dao.php [hunk 1]
```diff
@@ -64,7 +64,8 @@ public function save()
                 }
             }
 
-            Helper::upsert($this->db, self::TABLE_NAME, $data, $this->getPrimaryKey(self::TABLE_NAME));
+            $quoteIdentifiers = \Pimcore\Version::getMajorVersion() >= 11 ? true : false;
```
Comment (aryaantony92) [inline src/Model/SavedSearch/Dao.php]:
```suggestion
            $quoteIdentifiers = \Pimcore\Version::getMajorVersion() >= 11;
```

**Comment by aryaantony92** [comment]
@webdev-dp Thank you for the PR. Looks good except a small refactoring as suggested above.

**Review by aryaantony92 — APPROVED** [review]

**Comment by webdev-dp** [comment]
> @webdev-dp Thank you for the PR. Looks good except a small refactoring as suggested above.

done

**Pull request merged**